src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss

```diff
@@ -0,0 +1,4 @@
+.signaturePadding {
```

Comment:
can we not use came case for css class names. can you rename to cpc-signature-padding

Reply:
updated